ansible/idrsystems-playbooks/idr-analysis.yml

```diff
@@ -34,3 +34,9 @@
   - role: openmicroscopy.versioncontrol-utils
 
   - role: openmicroscopy.celery-docker
+
+- hosts: idr-analysis
+  tasks:
+  - file:
+      path: /scratch
+      mode: "1777"
```

Comment:
Choice is really up to you. Guess if we remove `/scratch` completely, should we use `state: absent` a la https://github.com/debops/ansible-lvm/blob/master/tasks/manage_lvm.yml?